editor/editor_resource_picker.cpp

```diff
@@ -624,7 +624,7 @@ void EditorResourcePicker::_get_allowed_types(bool p_with_convert, HashSet<Strin
 		if (p_with_convert) {
 			if (base == "BaseMaterial3D") {
 				p_vector->insert("Texture2D");
-			} else if (base == "ShaderMaterial") {
+			} else if (base == "ShaderMaterial" || base == "Material") {
```

Comment:
If anything, this is more robust:
```suggestion
			} else if (ClassDB::is_parent_class("ShaderMaterial", base)) {
```
Same should be done to Texture2D/ImageTexture below.

Reply:
> Same should be done to Texture2D/ImageTexture below.

Thanks!